.github/README.md

```diff
@@ -13,18 +13,15 @@ Lawnicons is best used on [Lawnchair 12.1](https://github.com/LawnchairLauncher/
 This project is mostly supported by our community. If you wish to contribute, check out the Contributing section below.
 
 ## Download
-See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build of Lawnicons.
-
-For the development versions with new icons, go to [the nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
+See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build. For debug builds with new icons go to [nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
```

Comment:
Regarding this, the `debug` builds should be called `development` builds. There are not a lot of *regular* users who know what "debug" builds are, so changing this could make it more clear what this sentence is about. If you want to keep the word "debug", you can use `development (debug)".

```suggestion
See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build. For development builds with new icons, go to [nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
```

Reply:
![Screenshot 2023-11-12 at 03 46 39](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/e33c044b-7416-4eae-bfae-ea1c66a2b94b)
I shortened this to fit everything into 1 line for most resolutions. But I agree that clearer is better than shorter.

![Screenshot 2023-11-12 at 03 50 54](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/ba945201-4281-43bc-a111-c932d4351b9f)
@SuperDragonXD There is a safer option, as the header gives context. How about this?

Reply:
Sure, that's fine.